frontend/package.json

```diff
@@ -39,6 +39,8 @@
     "@emotion/react": "^11.8.2",
     "@emotion/serialize": "^1.0.2",
     "@emotion/styled": "^11.8.1",
+    "@glideapps/glide-data-grid": "4.1.0",
```

Comment:
@tconkling or @AnOctopus , can you take a look at these libraries?

Reply:
Ken did a dependency check a minute ago. So, I think we are fine on the dependency side.